helm/nessie/templates/deployment.yaml

```diff
@@ -81,6 +81,35 @@ spec:
             {{- end }}
             {{- end }}
 
+            {{- if eq .Values.versionStoreType "TRANSACTIONAL" }}
```

Comment:
As TRANSACTIONAL backend store, Nessie supports H2 also along with Postgres.

So, Instead of checking with TRANSACTIONAL, can we have separate one for each transactional adapters? (similar to how non-transactional backend type as one for each). 

It will be easy to add some more transactional type support in the future.

Reply:
oh, I see that enum itself has just trasnactional, 

https://github.com/projectnessie/nessie/blob/main/servers/quarkus-common/src/main/java/org/projectnessie/quarkus/config/VersionStoreConfig.java#L35

@snazy, @nastra, @dimas-b:   Should we rename it as specific (like postgres) to support other transactional ones ?

Reply:
I'm not sure how this works but it seems this Nessie image is built only with Postgres: https://github.com/projectnessie/nessie/blob/main/servers/quarkus-cli/src/main/resources/application.properties#L26

Reply:
Yeah, I observed that and hence posted that question. 
@kerossin: Sorry for spamming your PR. I will keep it in a separate discussion.

Reply:
We have tests for H2 as well. If there's interest it should be possible to build an image for it too.

Reply:
but yes, at this point only Postgres can be selected at runtime. It looks like supporting other JDBC drivers will require some code changes... Please feel free to open an issue for this :) At this point H2 should be easy to enable, but other RDBMS' should be possible too.